cockatrice/src/tab_deck_editor.cpp

```diff
@@ -712,6 +719,18 @@ bool TabDeckEditor::actSaveDeck()
     return false;
 }
 
+bool TabDeckEditor::actExportDeck()
+{
+    DeckLoader *const deck = deckModel->getDeckList();
+    QString decklistUrl = deck->exportDeckToDecklist();
```

Comment:
Is it possible for `deck` to ever be null?

Reply:
Yes; null checking should be done

Reply:
Added